docs/reST/ref/transform.rst

```diff
@@ -243,6 +243,18 @@ Instead, always begin with the original image and scale to the desired size.)
 
    .. ## pygame.transform.average_color ##
 
+.. function:: grayscale
+
+   | :sl:`grayscale a surface`
+   | :sg:`grayscale(surface, dest_surface=None) -> Surface`
+
+   Grayscales the surface.
```

Comment:
Well, technically it returns a new surface that's been greyscaled from the original. But it would be also good to briefly describe the luminance formula you're using.